services/autoscaling/src/simcore_service_autoscaling/core/settings.py

```diff
@@ -60,7 +60,8 @@ class Config(EC2Settings.Config):
 class EC2InstancesSettings(BaseCustomSettings):
     EC2_INSTANCES_ALLOWED_TYPES: dict[str, EC2InstanceBootSpecific] = Field(
         ...,
-        description="Defines which EC2 instances are considered as candidates for new EC2 instance and their respective boot specific parameters",
+        description="Defines which EC2 instances are considered as candidates for new EC2 instance and their respective boot specific parameters"
+        "WARNING: if empty, all available ec2 instances are allowed",
```

Comment:
Hmm I think this warning is wrong. if empty none are allowed.

Reply:
Please double check

Reply:
Added validation to require at least one element